src/main/java/seedu/address/storage/JsonAdaptedProperty.java

```diff
@@ -87,21 +80,19 @@ public Property toModelType() throws IllegalValueException {
         }
         final Name modelName = new Name(name);
 
-        if (seller == null) {
-            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT, Seller.class.getSimpleName()));
+        if (address == null) {
+            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT, Address.class.getSimpleName()));
         }
-        if (!Seller.isValidSeller(seller)) {
-            throw new IllegalValueException(Seller.MESSAGE_CONSTRAINTS);
+        if (!Address.isValidAddress(address)) {
+            throw new IllegalValueException(Address.MESSAGE_CONSTRAINTS);
         }
-        final Seller modelSeller = new Seller(seller);
+        final Address modelAddress = new Address(address);
 
-        if (phone == null) {
-            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT, Phone.class.getSimpleName()));
-        }
-        if (!Phone.isValidPhone(phone)) {
-            throw new IllegalValueException(Phone.MESSAGE_CONSTRAINTS);
+        if (seller == null) {
+            throw new IllegalValueException(String.format(MISSING_FIELD_MESSAGE_FORMAT,
+                    JsonAdaptedPerson.class.getSimpleName()));
```

Comment:
Slight nitpick: Even though seller is of type `JsonAdaptedPerson`, would it be more suitable to have `Person.class` here?